feat: add rikky packages

mitosagi opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
Add all plugins from rikky to the xml.

**This PR depends on the following PR(s).**
- #69

## Related Issue
<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
The packages can be installed by apm on the main branch.

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Updated the modification date in `mod.xml`.

hal-shu-sato commented:
I found that rikky sets the version of the plugin internally.
![image](https://user-images.githubusercontent.com/28973861/146667442-d5be8309-596e-45a9-84ae-3a96b60005e6.png)

mitosagi commented:
Fixed the version of the plugins.
Also,
- Removed rikky/aulsPluginsByRikky from dependencies since it seems to work by itself.
- rikky/objectsound2 is removed because it is not confirmed to work yet. rikky/objectsound1 works fine.

hal-shu-sato reviewed (approved):
Nice data.
This works.
